website/package.json

```diff
@@ -24,6 +24,7 @@
     "@svgr/webpack": "^5.5.0",
     "classnames": "^2.3.1",
     "clsx": "^1.1.1",
+    "docusaurus": "^1.14.7",
```

Comment:
@vingov is this okay?

Reply:
Good call, We don't need this line because we are using docusaurus 2.x, which is added as a dependency on line no. 17. I removed this line and tested the blog page, it rendered correctly.

@yihua - Can you please remove this dependency?

Reply:
Yes, I removed it.  The dep is added when I build the website locally.